src/components/NoteDialogComponents/NoteDialogNote.scss

```diff
@@ -1,26 +1,51 @@
 @import "src/constants/style";
 @import "src/components/Note/Note.scss";
 
+$row-gap: 3px;
+
 .note-dialog__note {
   --note-footer-height: 28px;
 
   display: grid;
+  grid-template-areas:
+    "head head"
+    "content options"
+    "foot foot";
   grid-template-columns: 11fr 1fr;
-  grid-template-rows: 4fr 1fr;
+  grid-template-rows: 1fr 4fr 1fr;
+  row-gap: $row-gap;
 
   margin-bottom: $margin--default;
   padding: $padding--default $padding--medium;
   position: relative;
   border-radius: $note__border-radius;
   background-color: $color-white;
-  transition: transform 0.3s ease-in-out, box-shadow 0.3s ease-in-out;
+  transition:
+    transform 0.3s ease-in-out,
+    box-shadow 0.3s ease-in-out;
   list-style-type: none;
   z-index: $note-dialog-z-index;
   min-width: calc(240px);
   width: 100vw;
   max-width: 380px;
 }
 
+.note-dialog-note__header {
+  grid-area: head;
+}
+
+.note-dialog-note__main {
+  grid-area: content;
+}
```

Comment:
I found that an inserted image isn't scaled down

![grafik](https://github.com/inovex/scrumlr.io/assets/71148001/b90958c7-8018-41f6-8fea-0a924fcd61cc)

adding `overflow: auto` here fixes this
```suggestion
.note-dialog-note__main {
  grid-area: content;
  overflow: auto;
}
```

Reply:
thanks!